generator/metricbeat/{beat}/_meta/docker.yml

```diff
@@ -0,0 +1,4 @@
+{{.BeatName}}.config.modules:
+  path: ${path.config}/modules.d/*.yml
+  reload.enabled: false
+
```

Comment:
It'd be good if metricbeat would use the same files, so we remember to maintain them. Maybe a way to do it is to generate a custom beat called "metricbeat", and check that these files are the same as the ones for the real Metricbeat.

But let's forget about this by now.